sources/install/package_web.sh

```diff
@@ -723,6 +723,29 @@ function install_sqlmap() {
     add-to-list "sqlmap,https://github.com/sqlmapproject/sqlmap,Sqlmap is an open-source penetration testing tool that automates the process of detecting and exploiting SQL injection flaws"
 }
 
+function install_sqlisniper() {
+    colorecho "Installing SqliSniper"
+    git -C /opt/tools/ clone --depth 1 https://github.com/danialhalo/SqliSniper.git
+    python3 -m venv ./venv
+    ./venv/bin/python3 -m pip install -r requirements.txt
+    ln -s "/opt/tools/SqliSniper/sqlisniper.py" /opt/tools/bin/sqlisniper
+    add-history sqlisniper
+    add-test-command "sqlisniper -h"
+    add-to-list "sqlisniper,https://github.com/danialhalo/SqliSniper, SqliSniper is Advanced Time-based Blind SQL Injection fuzzer for HTTP Headers "
+}
+
+
+function install_sqlidetector() {
+    colorecho "Installing SQLiDetector"
+    git -C /opt/tools/ clone --depth 1 https://github.com/eslam3kl/SQLiDetector.git
+    python3 -m venv ./venv
+    ./venv/bin/python3 -m pip install -r requirements.txt
+    ln -s "/opt/tools/SQLiDetector/sqlidetector.py" /opt/tools/bin/sqlidetector
```

Comment:
Create an alias to call the tool

`alias sqlidetector.py='/opt/tools/SQLiDetector/venv/bin/python3 /opt/tools/SQLiDetector/sqlidetector.py'`